File: Modul6/C/PRAK605-2310817210020-Muhammad_Hasbiyan_Rusyadi.c
#include <stdio.h>
void main() {
    int ordo,baris1,baris2;
    printf("Masukkan jumlah ordo matriks : ");
    scanf("%d", &ordo);
    int arrA[ordo][ordo];
    int arrB[ordo][ordo];
    printf("Matriks A : \n");
    for (int i = 0 ; i < ordo ; i++) {
        for (int j = 0 ; j < ordo ; j++) {
        scanf("%d", &arrA[i][j]);}}
    printf("Matriks B : \n");
    for (int i = 0 ; i < ordo ; i++) {
        for(int j = 0 ; j < ordo ; j++) {
            scanf("%d", &arrB[i][j]);}}
    int hsl[ordo][ordo];
    for (int i = 0; i < ordo; i++) {
        for (int j = 0; j < ordo; j++) {
            hsl[i][j] = 0;
            for (int h = 0; h < ordo; h++) {
                hsl[i][j] += arrA[i][h] * arrB[h][j];}}}
    printf("Matriks AXB : \n");
    for (int i = 0; i < ordo; i++) {
        for (int j = 0; j < ordo; j++) {
            printf("%d ", hsl[i][j]);}
        printf("\n");}}

File: Modul6/Python/PRAK605-2310817210020-Muhammad_Hasbiyan_Rusyadi.py
def perkalianmats(A, B):
    n = len(A)
    result = [[0] * n for _ in range(n)]
    for i in range(n):
        for j in range(n):
            for k in range(n):
                result[i][j] += A[i][k] * B[k][j]
    return result
def print_matrix(matrix):
    for row in matrix:
        print(" ".join(map(str, row)))
n = int(input("Masukkan ordo matriks: "))
print("Masukkan elemen Matriks A:")
A = [list(map(int, input().split())) for _ in range(n)]
print("Masukkan elemen Matriks B:")
B = [list(map(int, input().split())) for _ in range(n)]
result_matrix = perkalianmats(A, B)
print("Matriks AXB:")
print_matrix(result_matrix)
print()